airbyte-integrations/connectors/source-gorgias-api/source_gorgias_api/gorgias_api.yaml

```diff
@@ -0,0 +1,36 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  requester:
+    url_base: "https://airbyte.gorgias.com"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "Basic {{ config['api_key'] }}"
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+    requester:
+      $ref: "*ref(definitions.requester)"
+  base_stream:
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+  ticket_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "tickets"
+      primary_key: "id"
+      path: "/api/tickets"
+
+streams:
+  - "*ref(definitions.ticket_stream)"
```

Comment:
There are a lot more streams to be implemented:
<img width="268" alt="image" src="https://user-images.githubusercontent.com/5154322/201432877-42e32317-f5c2-4ef5-ab52-4e4c2734707d.png">